en/messages.json

```diff
@@ -419,6 +419,15 @@
       }
     }
   },
+  "reopenInContainerShortcut": {
+    "message": "Reopen in Container $keyId$",
```

Comment:
Why "Container" uppercase?

Reply:
Thanks for the fast feedback @flodolo 

"Container" is capitalized to indicate that it is a significant term within the application.
Also it is capitalized in most of the cases across the file.

Reply:
You're right 🤔 (there are 5 of them with lowercase container though).

Keeping this one open until there is agreement in the main code repository.

Reply:
hey, @flodolo, have you discussed with the team?
btw, I still have an unreviewed PR [here](https://github.com/mozilla/multi-account-containers/pull/2721)

Reply:
I don't see any activity in the PR you linked to.

I've also raised that this workflow doesn't really work if there is no UX/content driving decisions, so we might need to change it.

Reply:
Sorry I didn't get what you mean. What should I do?

Reply:
Wait for a developer to comment in https://github.com/mozilla/multi-account-containers/pull/2721